File: Assignment_11/Assignment3.c
#include<stdio.h>

void Pattern(int iNo)
{
    int iCnt = 0;

    if(iNo > 0)
    {
        printf("Invalid Input\n");
    }
    
    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d * ",iCnt);
    }
    printf("\n");
}


int main()
{
    int iValue = 0;

    printf("Enter Number :");
    scanf("%d",&iValue);

    Pattern(iValue);


    return 0;
}

File: Assignment____47/Assignment3.java
// Input :
//  5   4   3   2   1
//  5   4   3   2   1
//  5   4   3   2   1

import java.util.*;

class Pattern
{
    public void Pattern(int Row, int Col)
    {
        for(int i = 1; i <= Row; i++)
        {
            for(int j = Col; j >= 1; j--)
            {
                System.out.print(j +"\t");
            }
            System.out.println();
        }
    }
}

class Assignment3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Number Of Rows :");
        int iValu1 = sobj.nextInt();

        System.out.println("Enter the Number Of Columns :");
        int iValu2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValu1,iValu2);

    }
}